app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetsController < ApplicationController
+  before_action :set_tweet, only: %i[show edit update destroy]
+  before_action :authenticate_user!, except: %i[index show]
+
+  # GET /tweets or /tweets.json
+  def index
+    @tweets = Tweet.all.order('created_at DESC')
+    @tweet = Tweet.new
+  end
+
+  # GET /tweets/1 or /tweets/1.json
+  def show; end
+
+  # GET /tweets/new
+  def new
+    @tweet = current_user.tweets.build
```

Comment:
it initializes an empty post @post = Post.new

Reply:
- Initialize an empty tweet: @tweet = Tweet.new